src/app/prt-staking/constants.ts

```diff
@@ -0,0 +1,12 @@
+import { getIndexTokenData } from '@indexcoop/tokenlists'
+
+import { ProductRevenueToken } from '@/app/prt-staking/types'
+
+export const prts: ProductRevenueToken[] = [
+  {
+    tokenData: getIndexTokenData('hyETH')!,
```

Comment:
Potentially could hardcode this in the future in tokenlists, e.g. `IndexEthereum.HYETH`

Reply:
Feel free to create a task in the tokenlists repo for discussion and with more infos on the approach.

Reply:
Added here: https://github.com/IndexCoop/tokenlists/issues/58